### PR TITLE
menu section for CL

### PR DESCRIPTION
**Pull request by mehulsatardekar**
Added menu section for CL 

(Note: Just check the sidebar)
Live Preview : https://framer-ui.netlify.app/docs/cards.html

### CODE REVIEW

File: components/menu/menu.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+.icon-nav-menu:hover{
+   background: #999a9b;
+   transition: background 0.4s ease-in-out;
+}
+
+.sidebar-menu{
+   display: block;
+   position: fixed;
+   height: 100vh;
+   background-color: #333333;
```
Comment (saurabhchirde) [inline components/menu/menu.css]: Try to use color variables

File: components/menu/menu.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+.selected-text{
+   padding: 10px 0px 10px 20px;
+   color: red;
+}
+.selected-text:hover{
+   background: #636364;
+}
+
+.selected-text:focus{
+   background: #636364;
```
Comment (saurabhchirde) [inline components/menu/menu.css]: Color variables here as well

**Review by saurabhchirde — APPROVED** [review]
Loved your menu section, it's clean and fresh. 
Overall code is well structured with proper naming conventions, looks good to merge.

File: components/menu/menu.css (hunk 2 above)
Comment (mehulsatardekar) [reply components/menu/menu.css]: will make the changes thanks for pointing out

File: components/menu/menu.css [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+   overflow-y: auto;
+    scroll-behavior: smooth;
+}
+
+.slider-content-section::-webkit-scrollbar{
+   width: 10px;
+}
+
+
+
```
Comment (gauravsinhaweb) [inline components/menu/menu.css:77]: remove unnecessary spacing.

File: docs/menu.html [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>FrameWork-UI</title>
+    <link rel="stylesheet" href="../components/menu/menu.css">
+    <meta name="theme-color" content="#2C2929">
+
+</head>
+
+<body>
+
+
```
Comment (gauravsinhaweb) [inline docs/menu.html]: remove unnecessary spacing.

File: docs/menu.html [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+                    <span class="font-white">Made 😎 by Mehul </span>
+                    
+                </div>
+             </div>
+        </aside>
+    </nav>
+    
+   
+    
+
```
Comment (gauravsinhaweb) [inline docs/menu.html]: remove unnecessary spacing.

**Review by gauravsinhaweb — APPROVED** [review]
Nice work!, Good to merge.

**Pull request merged**